options/posix/generic/pthread-stubs.cpp

```diff
@@ -863,23 +925,25 @@ int pthread_cond_destroy(pthread_cond_t *) {
 }
 
 int pthread_cond_wait(pthread_cond_t *__restrict cond, pthread_mutex_t *__restrict mutex) {
+	struct timespec *time;
```

Comment:
I think what you probably want to do here is allocate a struct on the stack, and then call ``pthread_cond_timedwait`` with ``&time``.